# Market Analysis for Profitable Apps on Google Play and App Store

Our aim for this project is to determine what type of apps will be the most profitable to develop. Our current business model are free apps with in-app ads, so we need to ensure that we build the type of apps that are most likely to attract a large number of users.

### Data Sets

We will be using the following data sets:
- [10,000 Android apps from Google Play](https://www.kaggle.com/lava18/google-play-store-apps)
- [7,000 iOS apps from the App Store](https://www.kaggle.com/ramamet4/app-store-apple-data-set-10k-apps)

### Exploring The Data Sets

In [50]:
from csv import reader

# Google Play
file = open("googleplaystore.csv", encoding="utf8")
read_file = reader(file)
android = list(read_file)
android_header = android[0]
android = android[1:]

# App Store
file = open("AppleStore.csv", encoding="utf8")
read_file = reader(file)
ios = list(read_file)
ios_header = ios[0]
ios = ios[1:]

To make it easier to explore the two data sets, we'll first write a function named `explore_data()` that we can use repeatedly to explore rows in a more readable way. We'll also add an option for our function to show the number of rows and columns for any data set.

In [51]:
def explore_data(dataset, start, end, rows_and_columns=False):
    dataset_slice = dataset[start:end]    
    for row in dataset_slice:
        print(row)
        print('\n') # adds a new (empty) line after each row

    if rows_and_columns:
        print('Number of rows:', len(dataset))
        print('Number of columns:', len(dataset[0]))

#### Android

In [52]:
print(android_header)
print("\n")
explore_data(android, 0, 3, True)

['App', 'Category', 'Rating', 'Reviews', 'Size', 'Installs', 'Type', 'Price', 'Content Rating', 'Genres', 'Last Updated', 'Current Ver', 'Android Ver']


['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


['Coloring book moana', 'ART_AND_DESIGN', '3.9', '967', '14M', '500,000+', 'Free', '0', 'Everyone', 'Art & Design;Pretend Play', 'January 15, 2018', '2.0.0', '4.0.3 and up']


['U Launcher Lite – FREE Live Cool Themes, Hide Apps', 'ART_AND_DESIGN', '4.7', '87510', '8.7M', '5,000,000+', 'Free', '0', 'Everyone', 'Art & Design', 'August 1, 2018', '1.2.4', '4.0.3 and up']


Number of rows: 10841
Number of columns: 13


From this we can see that this data set contains 10841 rows, and 13 columns. Some of the columns that we might be interested in are:
- App
- Category
- Rating
- Reviews
- Installs
- Type
- Price
- Genres
- Last updated

In [53]:
print(ios_header)
print("\n")
explore_data(ios, 0, 3, True)

['id', 'track_name', 'size_bytes', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', 'user_rating', 'user_rating_ver', 'ver', 'cont_rating', 'prime_genre', 'sup_devices.num', 'ipadSc_urls.num', 'lang.num', 'vpp_lic']


['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


['529479190', 'Clash of Clans', '116476928', 'USD', '0.0', '2130805', '579', '4.5', '4.5', '9.24.12', '9+', 'Games', '38', '5', '18', '1']


Number of rows: 7197
Number of columns: 16


From this we can see that this data set contains 7197 rows, and 16 columns. Some of the columns that we might be interested in are:
- track_name
- price
- rating_count_tot
- user_rating
- prime_genre

Some columns are not very descriptive but you can find more about these in the documentation [here](https://www.kaggle.com/ramamet4/app-store-apple-data-set-10k-apps/home)

### Data Cleaning

We need to clean the data sets to ensure we produce accurate results. This involves removing duplicates and looking for errors within our data set. Also, we are focused on only free apps, and target an English speaking audience. We will remove those that do not meet these requirements too.

In [54]:
# Check for rows that don't have all columns
for row in android:
    if len(row) != len(android_header):
        print(row)
        print(android.index(row))

['Life Made WI-Fi Touchscreen Photo Frame', '1.9', '19', '3.0M', '1,000+', 'Free', '0', 'Everyone', '', 'February 11, 2018', '1.0.19', '4.0 and up']
10472


In [55]:
del android[10472]

In [56]:
for row in ios:
    if len(row) != len(ios_header):
        print(row)
        print(ios.index(row))

Now let's check for duplicate entries

In [57]:
# Google Play duplicate entries
duplicate_apps = []
unique_apps = []

for row in android:
    name = row[0]
    if name in unique_apps:
        duplicate_apps.append(name)
    else:
        unique_apps.append(name)
        
print("Number of duplicate apps: {}".format(len(duplicate_apps)))
print("\n")
print("Examples of duplicate apps: {}".format(duplicate_apps[:10]))

Number of duplicate apps: 1181


Examples of duplicate apps: ['Quick PDF Scanner + OCR FREE', 'Box', 'Google My Business', 'ZOOM Cloud Meetings', 'join.me - Simple Meetings', 'Box', 'Zenefits', 'Google Ads', 'Google My Business', 'Slack']


In [58]:
# App Store duplicate entries
duplicate_apps = []
unique_apps = []

for row in ios:
    name = row[1]
    if name in unique_apps:
        duplicate_apps.append(name)
    else:
        unique_apps.append(name)
        
print("Number of duplicate apps: {}".format(len(duplicate_apps)))
print("\n")
print("Examples of duplicate apps: {}".format(duplicate_apps[:10]))

Number of duplicate apps: 2


Examples of duplicate apps: ['Mannequin Challenge', 'VR Roller Coaster']


Let's explore these entries a bit further since there is only 2 to deal with.

In [59]:
for row in ios:
    name = row[1]
    if name == "Mannequin Challenge" or name == "VR Roller Coaster":
        print(row)

['1173990889', 'Mannequin Challenge', '109705216', 'USD', '0.0', '668', '87', '3.0', '3.0', '1.4', '9+', 'Games', '37', '4', '1', '1']
['952877179', 'VR Roller Coaster', '169523200', 'USD', '0.0', '107', '102', '3.5', '3.5', '2.0.0', '4+', 'Games', '37', '5', '1', '1']
['1178454060', 'Mannequin Challenge', '59572224', 'USD', '0.0', '105', '58', '4.0', '4.5', '1.0.1', '4+', 'Games', '38', '5', '1', '1']
['1089824278', 'VR Roller Coaster', '240964608', 'USD', '0.0', '67', '44', '3.5', '4.0', '0.81', '4+', 'Games', '38', '0', '1', '1']


Upon further exploration of the the suspected duplicate rows we can see that quite a few data points differe. These include:
- id
- size_bytes
- rating_count_tot
- user_rating
- ver

So it's quite possible that these are different apps just with the same name.

#### Dealing with duplicates in Android data set

Rather than deleting these duplicate entries randomly, let's assume that the one with the highest reviews is the most recent and the one which we will keep within our data set.

To remove the duplicates, we will:

- Create a dictionary, where each dictionary key is a unique app name and the corresponding dictionary value is the highest number of reviews of that app.
- Use the information stored in the dictionary and create a new data set, which will have only one entry per app (and for each app, we'll only select the entry with the highest number of reviews).

In [60]:
# Google Play
reviews_max = {}

for row in android:
    name = row[0]
    num_of_reviews = float(row[3])
    
    if name in reviews_max and reviews_max[name] < num_of_reviews:
        reviews_max[name] = num_of_reviews
    elif name not in reviews_max:
        reviews_max[name] = num_of_reviews
        
print("Expected length: {}".format(len(android) - 1181)) # 1181 refers to num of duplicate apps in Google play data set
print("Length of reviews_max: {}".format(len(reviews_max)))

Expected length: 9659
Length of reviews_max: 9659


In [61]:
android_clean = []
already_added = []

for row in android:
    name = row[0]
    num_of_reviews = float(row[3])
    
    if name not in already_added and reviews_max[name] == num_of_reviews:
        android_clean.append(row)
        already_added.append(name)
        
print("Number of rows in android_clean: {}".format(len(android_clean)))

Number of rows in android_clean: 9659


### Dealing With Non-English Apps

In [62]:
def is_english_char(string):
    non_ascii = 0
    for char in string:
        if ord(char) > 127:
            non_ascii += 1
    if non_ascii > 3:
        return False
    else:
        return True

# Test function
print(is_english_char("Instagram"))
print(is_english_char("爱奇艺PPS -《欢乐颂2》电视剧热播"))
print(is_english_char("Instachat 😜"))

True
False
True


In [63]:
english_android_apps = []
english_ios_apps = []

for row in android_clean:
    name = row[0]
    if is_english_char(name):
        english_android_apps.append(row)

for row in ios:
    name = row[1]
    if is_english_char(name):
        english_ios_apps.append(row)
        
print("Number of english Android apps: {}".format(len(english_android_apps)))
print("Number of english iOS apps: {}".format(len(english_ios_apps)))

Number of english Android apps: 9614
Number of english iOS apps: 6183


### Removing Non Free Apps

In [64]:
free_english_android_apps = []
free_english_ios_apps = []

for row in english_android_apps:
    price = row[7]
    if price == "0":
        free_english_android_apps.append(row)
        
for row in english_ios_apps:
    price = row[4]
    if price == "0.0":
        free_english_ios_apps.append(row)
        
print(len(free_english_android_apps))
print(len(free_english_ios_apps))

8864
3222


We're left with 8864 Android apps and 3222 iOS apps, which should be enough for our analysis.

## App Validation

Our end goal is to add the app on both Google Play and the App Store, we need to find app profiles that are successful on both markets.

Once we have found a suitable app profile we will:

1. Build a minimal Android version of the app, and add it to Google Play.
1. If the app has a good response from users, we develop it further.
1. If the app is profitable after six months, we build an iOS version of the app and add it to the App Store.

Let's begin the analysis by getting a sense of what are the most common genres for each market.

### Most Common Genres

In [65]:
def freq_table(dataset, index):
    table = {}
    
    for row in dataset:
        key = row[index]
        if key in table:
            table[key] += 1
        else:
            table[key] = 1
            
    table_percentages = {}
    for key in table:
        percentage = (table[key] / len(dataset)) * 100
        table_percentages[key] = percentage 
    
    return table_percentages

def display_table(dataset, index):
    table = freq_table(dataset, index)
    table_display = []
    for key in table:
        key_val_as_tuple = (table[key], key)
        table_display.append(key_val_as_tuple)

    table_sorted = sorted(table_display, reverse = True)
    for entry in table_sorted:
        print(entry[1], ':', entry[0])

#### Android

In [66]:
display_table(free_english_android_apps, 1) # Category column

FAMILY : 18.907942238267147
GAME : 9.724729241877256
TOOLS : 8.461191335740072
BUSINESS : 4.591606498194946
LIFESTYLE : 3.9034296028880866
PRODUCTIVITY : 3.892148014440433
FINANCE : 3.7003610108303246
MEDICAL : 3.531137184115524
SPORTS : 3.395758122743682
PERSONALIZATION : 3.3167870036101084
COMMUNICATION : 3.2378158844765346
HEALTH_AND_FITNESS : 3.0798736462093865
PHOTOGRAPHY : 2.944494584837545
NEWS_AND_MAGAZINES : 2.7978339350180503
SOCIAL : 2.6624548736462095
TRAVEL_AND_LOCAL : 2.33528880866426
SHOPPING : 2.2450361010830324
BOOKS_AND_REFERENCE : 2.1435018050541514
DATING : 1.861462093862816
VIDEO_PLAYERS : 1.7937725631768955
MAPS_AND_NAVIGATION : 1.3989169675090252
FOOD_AND_DRINK : 1.2409747292418771
EDUCATION : 1.1620036101083033
ENTERTAINMENT : 0.9589350180505415
LIBRARIES_AND_DEMO : 0.9363718411552346
AUTO_AND_VEHICLES : 0.9250902527075812
HOUSE_AND_HOME : 0.8235559566787004
WEATHER : 0.8009927797833934
EVENTS : 0.7107400722021661
PARENTING : 0.6543321299638989
ART_AND_DESIGN : 

In [67]:
display_table(free_english_android_apps, -4) # Genre column

Tools : 8.449909747292418
Entertainment : 6.069494584837545
Education : 5.347472924187725
Business : 4.591606498194946
Productivity : 3.892148014440433
Lifestyle : 3.892148014440433
Finance : 3.7003610108303246
Medical : 3.531137184115524
Sports : 3.463447653429603
Personalization : 3.3167870036101084
Communication : 3.2378158844765346
Action : 3.1024368231046933
Health & Fitness : 3.0798736462093865
Photography : 2.944494584837545
News & Magazines : 2.7978339350180503
Social : 2.6624548736462095
Travel & Local : 2.3240072202166067
Shopping : 2.2450361010830324
Books & Reference : 2.1435018050541514
Simulation : 2.0419675090252705
Dating : 1.861462093862816
Arcade : 1.8501805054151623
Video Players & Editors : 1.7712093862815883
Casual : 1.7599277978339352
Maps & Navigation : 1.3989169675090252
Food & Drink : 1.2409747292418771
Puzzle : 1.128158844765343
Racing : 0.9927797833935018
Role Playing : 0.9363718411552346
Libraries & Demo : 0.9363718411552346
Auto & Vehicles : 0.9250902527075

From the analysis above we can see that the `genre` column provides finer details about the specific categorys but it is a bit noisy compared to the `category` column. So moving forward we will use the category column for a broader overview.

From the results we can see that `FAMILY`, `GAME` and `TOOLS` are the top 3 most common categories on Google Play.

#### iOS

In [68]:
display_table(free_english_ios_apps, -5)

Games : 58.16263190564867
Entertainment : 7.883302296710118
Photo & Video : 4.9658597144630665
Education : 3.662321539416512
Social Networking : 3.2898820608317814
Shopping : 2.60707635009311
Utilities : 2.5139664804469275
Sports : 2.1415270018621975
Music : 2.0484171322160147
Health & Fitness : 2.0173805090006205
Productivity : 1.7380509000620732
Lifestyle : 1.5828677839851024
News : 1.3345747982619491
Travel : 1.2414649286157666
Finance : 1.1173184357541899
Weather : 0.8690254500310366
Food & Drink : 0.8069522036002483
Reference : 0.5586592178770949
Business : 0.5276225946617008
Book : 0.4345127250155183
Navigation : 0.186219739292365
Medical : 0.186219739292365
Catalogs : 0.12414649286157665


Here we can see that there is a clear domination of entertainment focused apps on the App Store, it can also be noted that the second most popular is Entertainment. However, the fact that entertainment based apps are the most numerous doesn't also imply that they also have the greatest number of users — the demand might not be the same as the offer.

### Number of Installs per Genre

One way to find out what genres are the most popular (have the most users) is to calculate the average number of installs for each app genre. For the Google Play data set, we can find this information in the Installs column, but for the App Store data set this information is missing. As a workaround, we'll take the total number of user ratings.

In [69]:
# Function to help sort output for easier analysis
def print_sorted(table):
    table_display = []
    for key in table:
        key_val_as_tuple = (table[key], key)
        table_display.append(key_val_as_tuple)
        
    table_sorted = sorted(table_display, reverse = True)
    for entry in table_sorted:
        print(entry[1], ':', entry[0])

#### Google Play

In [70]:
categories_android = freq_table(free_english_android_apps, 1)
category_installs = {}
for category in categories_android:
    total = 0
    len_category = 0
    for app in free_english_android_apps:
        category_app = app[1]
        if category_app == category:            
            num_of_installs = app[5]
            num_of_installs = num_of_installs.replace(',', '')
            num_of_installs = num_of_installs.replace('+', '')
            total += float(num_of_installs)
            len_category += 1
    avg_num_of_installs = total / len_category
    category_installs[category] = avg_num_of_installs
    
print_sorted(category_installs)

COMMUNICATION : 38456119.167247385
VIDEO_PLAYERS : 24727872.452830188
SOCIAL : 23253652.127118643
PHOTOGRAPHY : 17840110.40229885
PRODUCTIVITY : 16787331.344927534
GAME : 15588015.603248259
TRAVEL_AND_LOCAL : 13984077.710144928
ENTERTAINMENT : 11640705.88235294
TOOLS : 10801391.298666667
NEWS_AND_MAGAZINES : 9549178.467741935
BOOKS_AND_REFERENCE : 8767811.894736841
SHOPPING : 7036877.311557789
PERSONALIZATION : 5201482.6122448975
WEATHER : 5074486.197183099
HEALTH_AND_FITNESS : 4188821.9853479853
MAPS_AND_NAVIGATION : 4056941.7741935486
FAMILY : 3695641.8198090694
SPORTS : 3638640.1428571427
ART_AND_DESIGN : 1986335.0877192982
FOOD_AND_DRINK : 1924897.7363636363
EDUCATION : 1833495.145631068
BUSINESS : 1712290.1474201474
LIFESTYLE : 1437816.2687861272
FINANCE : 1387692.475609756
HOUSE_AND_HOME : 1331540.5616438356
DATING : 854028.8303030303
COMICS : 817657.2727272727
AUTO_AND_VEHICLES : 647317.8170731707
LIBRARIES_AND_DEMO : 638503.734939759
PARENTING : 542603.6206896552
BEAUTY : 51315

From this data we can see that `COMMUNICATION` type apps have the most users, this is probably due to popular apps like Whatsapp, Facebook messenger, Telegram etc. `VIDEO_PLAYERS` follows a similar trend with apps like Youtube, Google Play Movies & TV, or MX Player. `SOCIAL` is another category dominated by giants like Facebook, Snapchat etc. The next popular categories are a bit more interesting. `PRODUCTIVITY` would be a good choice, since we are interested in building these type of apps, but we do have to keep in mind that it also contains some massive brands like Evernote, Dropbox etc.

#### App Store

In [71]:
genres_ios = freq_table(free_english_ios_apps, -5)
genre_ratings = {}
for genre in genres_ios:
    total = 0
    len_genre = 0
    for row in free_english_ios_apps:
        genre_app = row[-5]
        if genre_app == genre:
            num_of_installs = float(row[5])
            total += num_of_installs
            len_genre += 1
    avg_user_ratings = total/len_genre
    genre_ratings[genre] = avg_user_ratings
    
print_sorted(genre_installs)

Navigation : 86090.33333333333
Reference : 74942.11111111111
Social Networking : 71548.34905660378
Music : 57326.530303030304
Weather : 52279.892857142855
Book : 39758.5
Food & Drink : 33333.92307692308
Finance : 31467.944444444445
Photo & Video : 28441.54375
Travel : 28243.8
Shopping : 26919.690476190477
Health & Fitness : 23298.015384615384
Sports : 23008.898550724636
Games : 22788.6696905016
News : 21248.023255813954
Productivity : 21028.410714285714
Utilities : 18684.456790123455
Lifestyle : 16485.764705882353
Entertainment : 14029.830708661417
Business : 7491.117647058823
Education : 7003.983050847458
Catalogs : 4004.0
Medical : 612.0


Surprisingly we get a different result on the App Store. Naviagtion apps seem to be the most popular, which again could be due to the very popular Google Maps and Waze apps. It would be tough to build an app that would compete with these, so navigation is not something we will look further into.

Here our top choice from the Google Play store (`PRODUCTIVITY`) is lower in popularity but still has some decent volume compared to other genres.

# Conclusion

Having a look at the analysis, we have chosen a few genres/categories that meet our needs, as well as personal interest:

- Productivity
- Books
- Weather
- News
- Health and fitness


In [72]:
# Top 50 apps for productivity
num_of_rows = 1

for row in free_english_android_apps:
    if row[1] == "PRODUCTIVITY" and num_of_rows < 50:
        print("{} : {}".format(row[0], row[5]))
        num_of_rows += 1

Microsoft Word : 500,000,000+
All-In-One Toolbox: Cleaner, Booster, App Manager : 10,000,000+
AVG Cleaner – Speed, Battery & Memory Booster : 10,000,000+
QR Scanner & Barcode Scanner 2018 : 10,000,000+
Chrome Beta : 10,000,000+
Microsoft Outlook : 100,000,000+
Google PDF Viewer : 10,000,000+
My Claro Peru : 5,000,000+
Power Booster - Junk Cleaner & CPU Cooler & Boost : 1,000,000+
Google Assistant : 10,000,000+
Microsoft OneDrive : 100,000,000+
Calculator - unit converter : 50,000,000+
Microsoft OneNote : 100,000,000+
Metro name iD : 10,000,000+
Google Keep : 100,000,000+
Archos File Manager : 5,000,000+
ES File Explorer File Manager : 100,000,000+
ASUS SuperNote : 10,000,000+
HTC File Manager : 10,000,000+
MyMTN : 1,000,000+
Dropbox : 500,000,000+
ASUS Quick Memo : 10,000,000+
HTC Calendar : 10,000,000+
Google Docs : 100,000,000+
ASUS Calling Screen : 10,000,000+
lifebox : 5,000,000+
Yandex.Disk : 5,000,000+
Content Transfer : 5,000,000+
HTC Mail : 10,000,000+
Advanced Task Killer : 50

In [73]:
# Top 50 apps for books
num_of_rows = 1

for row in free_english_android_apps:
    if row[1] == "BOOKS_AND_REFERENCE" and num_of_rows < 50:
        print("{} : {}".format(row[0], row[5]))
        num_of_rows += 1

E-Book Read - Read Book for free : 50,000+
Download free book with green book : 100,000+
Wikipedia : 10,000,000+
Cool Reader : 10,000,000+
Free Panda Radio Music : 100,000+
Book store : 1,000,000+
FBReader: Favorite Book Reader : 10,000,000+
English Grammar Complete Handbook : 500,000+
Free Books - Spirit Fanfiction and Stories : 1,000,000+
Google Play Books : 1,000,000,000+
AlReader -any text book reader : 5,000,000+
Offline English Dictionary : 100,000+
Offline: English to Tagalog Dictionary : 500,000+
FamilySearch Tree : 1,000,000+
Cloud of Books : 1,000,000+
Recipes of Prophetic Medicine for free : 500,000+
ReadEra – free ebook reader : 1,000,000+
Anonymous caller detection : 10,000+
Ebook Reader : 5,000,000+
Litnet - E-books : 100,000+
Read books online : 5,000,000+
English to Urdu Dictionary : 500,000+
eBoox: book reader fb2 epub zip : 1,000,000+
English Persian Dictionary : 500,000+
Flybook : 500,000+
All Maths Formulas : 1,000,000+
Ancestry : 5,000,000+
HTC Help : 10,000,000+
E

In [74]:
# Top 50 apps for weather
num_of_rows = 1

for row in free_english_android_apps:
    if row[1] == "WEATHER" and num_of_rows < 50:
        print("{} : {}".format(row[0], row[5]))
        num_of_rows += 1

The Weather Channel: Rain Forecast & Storm Alerts : 50,000,000+
Weather forecast : 1,000,000+
AccuWeather: Daily Forecast & Live Weather Reports : 50,000,000+
Live Weather Pro : 10,000+
Weather by WeatherBug: Forecast, Radar & Alerts : 10,000,000+
weather - weather forecast : 1,000,000+
MyRadar NOAA Weather Radar : 10,000,000+
SMHI Weather : 1,000,000+
Free live weather on screen : 1,000,000+
Weather Radar Widget : 1,000,000+
Weather –Simple weather forecast : 10,000,000+
Weather Crave : 5,000,000+
Klara weather : 500,000+
Yahoo Weather : 10,000,000+
Real time Weather Forecast : 1,000,000+
METEO FRANCE : 5,000,000+
APE Weather ( Live Forecast) : 5,000,000+
Live Weather & Daily Local Weather Forecast : 1,000,000+
Weather : 10,000,000+
Rainfall radar - weather : 5,000,000+
Yahoo! Weather for SH Forecast for understanding the approach of rain clouds Free : 1,000,000+
The Weather Network : 5,000,000+
Klart.se - Sweden's best weather : 1,000,000+
GO Weather - Widget, Theme, Wallpaper, Effic

In [75]:
# Top 50 apps for news
num_of_rows = 1

for row in free_english_android_apps:
    if row[1] == "NEWS_AND_MAGAZINES" and num_of_rows < 50:
        print("{} : {}".format(row[0], row[5]))
        num_of_rows += 1

Fox News – Breaking News, Live Video & News Alerts : 10,000,000+
NEW - Read Newspaper, News 24h : 10,000,000+
BaBe + - Indonesian News : 1,000,000+
daily News : 1,000,000+
BBC News : 10,000,000+
Free TV Shows App:News, TV Series, Episode, Movies : 1,000,000+
News24 : 1,000,000+
Le Monde, the continuous news : 5,000,000+
Wireless news : 1,000,000+
CNN Breaking US & World News : 10,000,000+
Gnoche entertainment news · sports news is also free : 5,000,000+
BaBe - Read News : 10,000,000+
Nigeria News NAIJ.com : 1,000,000+
detikcom - Latest & Most Complete News : 10,000,000+
Dailyhunt (Newshunt) - Latest News, Viral Videos : 50,000,000+
BaBe Lite - Read Quota Saving News : 1,000,000+
ARY NEWS URDU : 500,000+
Bengali Newspaper - The first L. : 1,000,000+
Read- Latest News, Information, Gossip and Politics : 10,000,000+
Reddit: Social News, Trending Memes & Funny Videos : 10,000,000+
BBC Persian | BBC Farsi News : 100,000+
Opera News - Trending news and videos : 10,000,000+
Topbuzz: Breaking 

In [76]:
# Top 50 apps for Health & Fitness
num_of_rows = 1

for row in free_english_android_apps:
    if row[1] == "HEALTH_AND_FITNESS" and num_of_rows < 50:
        print("{} : {}".format(row[0], row[5]))
        num_of_rows += 1

Step Counter - Calorie Counter : 500,000+
Lose Belly Fat in 30 Days - Flat Stomach : 5,000,000+
Pedometer - Step Counter Free & Calorie Burner : 1,000,000+
Six Pack in 30 Days - Abs Workout : 10,000,000+
Lose Weight in 30 Days : 10,000,000+
Pedometer : 10,000,000+
LG Health : 10,000,000+
Step Counter - Pedometer Free & Calorie Counter : 10,000,000+
Pedometer, Step Counter & Weight Loss Tracker App : 10,000,000+
Sportractive GPS Running Cycling Distance Tracker : 1,000,000+
30 Day Fitness Challenge - Workout at Home : 10,000,000+
Home Workout for Men - Bodybuilding : 1,000,000+
Fat Burning Workout - Home Weight lose : 100,000+
Buttocks and Abdomen : 500,000+
Walking for Weight Loss - Walk Tracker : 100,000+
Running & Jogging : 500,000+
Sleep Sounds : 1,000,000+
Fitbit : 10,000,000+
Lose Belly Fat-Home Abs Fitness Workout : 50,000+
Cycling - Bike Tracker : 500,000+
Abs Training-Burn belly fat : 100,000+
Calorie Counter - EasyFit free : 1,000,000+
Aunjai i lert u : 500,000+
Garmin Connect